Automattic/jetpack
# Boost: add yellow color for ISA potential size pills

Fixes https://github.com/Automattic/jetpack/issues/31739

## Proposed changes:

* Update potential size suggestion pills to include yellow (to match designs p1HpG7-lar-p2).
* The yellow color is added for optimizations that are less than 30% of the original size.

<img width="672" alt="CleanShot 2023-07-07 at 17 30 15@2x" src="https://github.com/Automattic/jetpack/assets/11799079/dee6ef1b-124f-49c0-a9bb-3634c26628d7">


### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup this version of Boost and enable Image Size Analysis;
* Request a new report or open an existing one;
* Make sure "potential size" column includes a yellow pill.
